readme.md

```diff
@@ -400,7 +400,7 @@ test.only('will be run', t => {
 });
 ```
 
-`.only` applies across all test files, so if you use it in one file, no tests from the other file will run.
+*Note:* The `.only` modifier applies to the test file it's defined in, so if you run multiple test files, tests in other files will stil run. If you want to only run the `test.only` test, specify just that test file to AVA.
```

Comment:
Typo in `stil run`.

Rather than `specify just that test file to AVA`, say `provide just that test file to AVA`?